### PR TITLE
Add tomte HK tags

### PR DESCRIPTION
**Pull request by fevac**
#### Added
- HK tags for tomte 

### How to prepare for test
- [ ] ssh to hasta.scilifelab.se
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_hermes -t hermes -b tomte_store`

### How to test
- [ ] Do ...

### Expected test outcome
- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

File: cg_hermes/config/tomte.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+mandatory by default. However, the tags that are available to a particular analysis is mandatory for that analysis.
+"""
+
+from cg_hermes.config.nextflow import NEXTFLOW_TAGS
+
+TOMTE_COMMON_TAGS = {
+    frozenset({"fraser", "fraser-clinical"}): {
+        "is_mandatory": False,
+        "tags": ["fraser", "clinical"],
+        "used_by": ["deliver"],
```
Comment (ivadym) [inline cg_hermes/config/tomte.py:13]:
```suggestion
        "used_by": ["clinical-delivery"],
```

File: cg_hermes/config/tomte.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    },
+    frozenset({"fraser", "fraser-research"}): {
+        "is_mandatory": False,
+        "tags": ["fraser", "research"],
+        "used_by": ["deliver"],
```
Comment (ivadym) [inline cg_hermes/config/tomte.py:18]: Have we identified which ones we're keeping for long term storage?

Comment (fevac) [reply cg_hermes/config/tomte.py:18]: not yet, I will ask and add `long-term-storage` for those

Comment (ivadym) [reply cg_hermes/config/tomte.py:18]: Sweet 🍬

**Review by ivadym — APPROVED** [review]
Nice! Just add the `long-term-storage` tag to the protected tags for Tome when you figure out the list (doesn't have to be in this PR)

File: cg_hermes/config/tomte.py (hunk 1 above)
Comment (fevac) [reply cg_hermes/config/tomte.py]: added

File: cg_hermes/config/tomte.py (hunk 2 above)
Comment (fevac) [reply cg_hermes/config/tomte.py]: added!

**Comment by fevac** [comment]
> Just came to my mind that we should also add some Tomte specific pytests for export, convert, validate, etc. @fevac

@ivadym just in time! added!

**Comment by fevac** [comment]
- deployed
image
image

**Pull request merged**